postprocessing/janus-pp-rec.c

```diff
@@ -1488,3 +1575,30 @@ static gint janus_pp_skew_compensate_audio(janus_pp_frame_packet *pkt, janus_pp_
 
 	return exit_status;
 }
+
+static double get_latency(const janus_pp_frame_packet *tmp, int rate) {
+	/* Get latency of packet based on time of arrival (at server side) and the RTP timestamp */
+	double corrected_ts = tmp->ts-list->ts;
+	double rts = corrected_ts/(double) rate;
+	double pts = (double) tmp->p_ts/1000;
+	return pts-rts;
+}
+
+static double get_moving_average_of_latency(janus_pp_frame_packet *pkt, int rate, int num_of_packets) {
+	/* Get a moving average of packet latency using the get_latency function */
+	if(!pkt) {
+		return 0;
+	}
+
+	int packets = 0;
+	double sum = 0;
+
+	janus_pp_frame_packet *tmp = pkt;
+	while(tmp && packets < num_of_packets) {
+		sum += get_latency(tmp, rate);
+		tmp = tmp->prev;
+		packets++;
+	}
+
+	return sum/packets;
+}
```

Comment:
Looks good to me too, the only thing I noticed is a missing new line here, can you fix that?

Reply:
Done.